src/commcare_cloud/fab/deploy_diff.py

```diff
@@ -51,14 +53,26 @@ def deployed_commit_matches_latest_commit(self):
             return self.current_commit == self.deploy_commit or long.startswith(short)
         return False
 
+    def get_changelogs(self):
+        def get_commit_date(commit):
+            return datetime.strptime(
+                self.repo.get_commit(commit).last_modified,
+                "%a, %d %b %Y %H:%M:%S GMT"
+            )
+        return get_changelogs_in_date_range(
+            get_commit_date(self.current_commit),
+            get_commit_date(self.deploy_commit)
```

Comment:
It doesn't look like there is a way in Github API to get both the commits in one request, so making two separate requests.